cmake/CmakeFunctionHelper.cmake

```diff
@@ -842,7 +848,10 @@ else() # default cpu backend
       add_gemm_configuration(
         "${data}"  64 "false" "false" "false"
         64 8 8 8 8 1 1 1 1 1 1 1 1 1 float float "no_local" "standard" "partial" 1 "strided" "false" "false")
-    endif()
+      add_gemm_configuration(
```

Comment:
I think you will need to update the `add_gemm_configuration` calls for all target hardwares to support `symm_a` and `symm_b` template parameters for those hardware.

Reply:
I have added that in the loop above. Similar of what we do for `trans_a` and `trans_b`.